### PR TITLE
Feature/new selenoid capabilities

### PR DESCRIPTION
**Pull request by markeloff86**
Have been added:
1) new capabilities for Selenoid
2) Opera support for Selenoid
3) condition for maximize browser
4) function for changing default browser size

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java [hunk 1]
```diff
@@ -90,6 +91,7 @@
     private static final String WEBDRIVER_BROWSER_VERSION = Props.get("webdriver.browser.version");
     private static final String WEBDRIVER_OS_ARCHITECTURE = Props.get("webdriver.os.arch");
     private static final String WEBDRIVER_BROWSER_PATH = Props.get("webdriver.browser.path");
+    private static final String WEBDRIVER_BROWSER_SIZE = Props.get("webdriver.browser.size");
```
Comment (0I0u3uk) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: Please add info about this param to https://github.com/sbtqa/page-factory/blob/master/src/main/resources/config/application.properties-sample

File: src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java [hunk 2]
```diff
@@ -3,85 +3,110 @@
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.drivers.TagWebDriver;
 import ru.sbtqa.tag.qautils.properties.Props;
 
+import java.util.HashMap;
+
+import static org.openqa.selenium.remote.BrowserType.OPERA;
```
Comment (0I0u3uk) [inline src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java]: We're using default import pattern, without newline

File: src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java [hunk 3]
```diff
@@ -3,85 +3,110 @@
...
+    private static final String SELENOID_SCREEN_RESOLUTION = Props.get("selenoid.screenResolution");
+    private static final String SELENOID_ENABLE_VIDEO = Props.get("selenoid.enableVideo");
+    private static final String SELENOID_VIDEO_NAME = Props.get("selenoid.video.name");
+    private static final String SELENOID_VIDEO_SCREEN_SIZE = Props.get("selenoid.video.screenSize");
+    private static final String SELENOID_VIDEO_FRAME_RATE = Props.get("selenoid.video.frameRate");
+    private static final String SELENOID_NAME_OF_TESTS = Props.get("selenoid.nameOfTests");
+    private static final String SELENOID_TIME_ZONE = Props.get("selenoid.timeZone");
+    private static final String SELENOID_HOST_ENTRIES = Props.get("selenoid.hostEntries");
+    private static final String SELENOID_APPLICATION_CONTAINERS = Props.get("selenoid.applicationContainers");
+    private static final String SELENOID_CONTAINER_LABLES = Props.get("selenoid.containerLables");
```
Comment (0I0u3uk) [inline src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java]: Also add this options to https://github.com/sbtqa/page-factory/blob/master/src/main/resources/config/application.properties-sample with description, or link to selenoid doc

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java (hunk 1 above)
Comment (markeloff86) [reply src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: Done

File: src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java (hunk 3 above)
Comment (markeloff86) [reply src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java]: Done

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java [hunk 4]
```diff
@@ -78,7 +79,7 @@
     private static final String WEBDRIVER_STARTING_URL = Props.get("webdriver.starting.url");
     private static final String WEBDRIVER_PROXY = Props.get("webdriver.proxy");
     private static final boolean WEBDRIVER_BROWSER_IE_KILL_ON_DISPOSE = Boolean.parseBoolean(Props.get("webdriver.browser.ie.killOnDispose", "false"));
-    private static final String WEBDRIVER_BROWSER_NAME = Props.get("webdriver.browser.name").equalsIgnoreCase(IE_BROWSER_TYPE)
+    public static final String WEBDRIVER_BROWSER_NAME = Props.get("webdriver.browser.name").equalsIgnoreCase(IE_BROWSER_TYPE)
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: Why are you make WEBDRIVER_BROWSER_NAME public? We have getBrowserName method.

**Pull request merged**